adhesive/telegram_bot.py

```diff
@@ -127,7 +127,6 @@ async def maybe_enter_convo(event, is_link, response):
 			tags=[],
 			nsfw=False,
 			original=False,
-			animated=False,
```

Comment:
Animation detection is done by the backend, so no need to specify it here

Reply:
Add that as a comment then

Reply:
:heavy_check_mark:

Reply:
I do not see said change.

Reply:
Nevermind.